src/dex-ui/pages/dao/CreateADAO/forms/NFTDAOReviewForm.tsx

```diff
@@ -97,7 +97,7 @@ export function NFTDAOReviewForm() {
             id: "voting.duration",
             label: "VOTING DURATION",
             type: "number",
-            unit: "Days",
+            unit: "Blocks",
```

Comment:
`Blocks` isn't a universally understood measurement of time from the user's perspective. Is there any way to estimate the duration of blocks?

Reply:
@damiano-melcarne-lab49  I have addressed the PR feedbacks on separate PR, let me know if you have any feedbacks